nixos/modules/services/backup/sanoid.nix

```diff
@@ -0,0 +1,163 @@
+{ config, stdenv, lib, pkgs, ... }:
```

Comment:
`stdenv` can and should be removed, it doesn't get passed in the module system (it would throw an error if accessed)